### DISCUSSION TITLE
Transforming images via CLI

### DISCUSSION THREAD

**Discussion by lqze** [parcel-bundler/parcel#5544] [Q&A] [answered]
Is it possible in Parcel V2 to transform and optimise images into `.webp` formats?

My `.parcelrc` is as follows:
```
{
  "extends": "@parcel/config-default",
  "transformers": {
    "*.{jpg,png,svg}": ["@parcel/transformer-image"]
  }
}
```

My command would be: `parcel build 'app/images/*.*' --dist-dir './assets/images/'`.

Currently this does no optimisation or transform.

I've scoured the documentation but can't really find any clear information on how to achieve this.

Thank you!

**Comment by mischnic** [comment, marked as answer]
By default, that transformer doesn't do anything. Only if you specify some options: https://v2.parceljs.org/recipes/image/

And setting these query parameters from the CLI for the entries isn't possible at the moment.

**Reply by DeMoorJasper** [reply]
The transformer's aim is also not to optimise, it's to transform. However converting a png to a webp image will inevitably optimise the image quite a lot...

I think we will also release an optimiser for images at some point, just not yet sure which library we should use. Personally I'm interested in using `squoosh` by the google chrome team once they make a JavaScript api, we could probably even contribute that ourselves...

**Comment by lqze** [comment]
So I know the imagemin plug-in exists for parcel v1, is it intended to work on v2 as well?

Is supporting CLI arguments for the image-transformer in the intended features?
